src/modules/filesystem/DroppedFile.cpp

```diff
@@ -173,12 +190,22 @@ int64 DroppedFile::tell()
 	if (file == nullptr)
 		return -1;
 
-	return (int64) ftell(file);
+#ifdef LOVE_WINDOWS
+	return (int64) _ftelli64(file);
+#else
+	return (int64) ftello(file);
+#endif
 }
 
 bool DroppedFile::seek(uint64 pos)
 {
-	return file != nullptr && fseek(file, (long) pos, SEEK_SET) == 0;
+	return file != nullptr &&
+
+#ifdef LOVE_WINDOWS
+	_fseeki64(file, (int64) pos, SEEK_SET) == 0;
+#else
+	fseeko(file, (off_t) pos, SEEK_SET) == 0;
+#endif
```

Comment:
```suggestion
	if (file == nullptr)
		return false;

#ifdef LOVE_WINDOWS
	return _fseeki64(file, (int64) pos, SEEK_SET) == 0;
#else
	return fseeko(file, (off_t) pos, SEEK_SET) == 0;
#endif
```

Splitting this `return` on multiple lines feels kinda unexpected (to me)

Reply:
I think you're right.